tsp-output/@azure-tools/typespec-csharp/src/Generated/OpenAIClient.cs

```diff
@@ -46,100 +46,20 @@ public OpenAIClient(Uri endpoint, ApiKeyCredential credential, OpenAIClientOptio
             options ??= new OpenAIClientOptions();
 
             _keyCredential = credential;
-            _pipeline = ClientPipeline.Create(options, Array.Empty<PipelinePolicy>(), new PipelinePolicy[] { ApiKeyAuthenticationPolicy.CreateHeaderApiKeyPolicy(_keyCredential, AuthorizationHeader, AuthorizationApiKeyPrefix) }, Array.Empty<PipelinePolicy>());
+            var authenticationPolicy = ApiKeyAuthenticationPolicy.CreateBearerAuthorizationPolicy(_keyCredential);
+            _pipeline = ClientPipeline.Create(options,
+                perCallPolicies: ReadOnlySpan<PipelinePolicy>.Empty,
+                perTryPolicies: new PipelinePolicy[] { authenticationPolicy },
+                beforeTransportPolicies: ReadOnlySpan<PipelinePolicy>.Empty);
             _endpoint = endpoint;
         }
 
         private Audio _cachedAudio;
-        private Assistants _cachedAssistants;
```

Comment:
These are just deleted because I'm illustrating MPFD just for the audio endpoints.  There are also OpenAI MPFD endpoints for Files and Images.  Let me know if you need more than what's here to generate those.